cfpb/sbl-frontend
# feat(home): add privacy, consent, and PRA notices

# **Waiting for design review before code review**
#### Design might change from this version, but this PR will be used to review design with @natalia-fitzgerald once we get her env set up

Add usage consent, privacy, and paperwork reduction act (PRA) notices to home

Completes Tasks: https://github.com/cfpb/sbl-project/issues/41, https://github.com/cfpb/sbl-project/issues/42, https://github.com/cfpb/sbl-project/issues/43

Completes Story: https://github.com/cfpb/sbl-project/issues/29

## TODO

- use the TextIntroduction component when it's ready
- write Cypress tests once design review is done

## Changes

- adds a notice and consent well to home
- adds a page for Paperwork Reduction Act notice
- adds a page for Privacy Act notice

## How to test this PR

1. Is the notice and consent visible on the main page?
2. Is the Paperwork Reduction Act notice page navigable from the sidebar and visible?
3. Is the Privacy Act notice page navigable from the sidebar and visible?

## Screenshots
<img width="1466" alt="Screenshot 2023-10-12 at 5 21 10 AM" src="https://github.com/cfpb/sbl-frontend/assets/19983248/aec53af4-626d-4ca8-880a-82ab7529efac">

<img width="1034" alt="Screenshot 2023-10-12 at 5 20 39 AM" src="https://github.com/cfpb/sbl-frontend/assets/19983248/c23e2110-4412-49ce-86da-7c93755e19f6">

<img width="1033" alt="Screenshot 2023-10-12 at 5 20 15 AM" src="https://github.com/cfpb/sbl-frontend/assets/19983248/9f04ac2b-ed3a-48f1-a82f-9e527cece4bb">
